react/Autosuggest/Autosuggest.js

```diff
@@ -70,16 +71,26 @@ export default class Autosuggest extends Component {
     }
   }
 
-  renderSuggestionsContainer({ containerProps, children }) {
+  renderSuggestionsContainer = ({ containerProps, children }) => {
     const { ref, ...rest } = containerProps;
+    const { showMobileBackdrop } = this.props;
+    const areSuggestionsShown = children !== null;
     const callRef = isolatedScroll => {
       if (isolatedScroll !== null) {
         ref(isolatedScroll.component);
       }
     };
 
     return (
-      <IsolatedScroll {...rest} ref={callRef} children={children} />
+      <IsolatedScroll {...rest} ref={callRef}>
+        {children}
+        {
+          areSuggestionsShown &&
+          smallDeviceOnly() &&
+          showMobileBackdrop ?
```

Comment:
Super minor, but maybe change the order of this to short circuit on `showMobileBackdrop` first?